rosbag2_storage/src/rosbag2_storage/impl/storage_factory_impl.hpp

```diff
@@ -17,8 +17,10 @@
 #define ROSBAG2_STORAGE__IMPL__STORAGE_FACTORY_IMPL_HPP_
 
 #include <algorithm>
+#include <vector>
 #include <memory>
 #include <string>
+#include <unordered_set>
```

Comment:
```suggestion
#include <algorithm>
#include <memory>
#include <string>
#include <unordered_set>
#include <vector>
```

Reply:
Fixed in the https://github.com/ros2/rosbag2/pull/1881/commits/505035c289f76d66d030946ab6146427c6827d1a